lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -208,9 +208,14 @@ def test_not_authorized(self):
         self.assertEqual(response.status_code, 404)
 
 
-class CourseListTests(CourseViewTestsMixin, ModuleStoreTestCase):
+class CourseListTests(CourseViewTestsMixin, SharedModuleStoreTestCase):
     view = 'course_structure_api:v0:list'
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
By the way, I thought `super()` traverses through all parent base classes.  The order in which the Mixins and the `SharedModuleStoreTestCase` appear in the class list will dictate whether the setUpClass gets called.